src/styles/_workspace.scss

```diff
@@ -106,9 +108,22 @@ $code-color-error: #ff4444;
 
     .Editor {
       flex: 1 1 auto;
+      margin: 0 0.5rem 0 0.5rem;
+      padding: 0;
     }
   }
 
+  .side-content-header {
```

Comment:
This needs to wrap when overflowing (for mobile users / moving the resizable editor). I'll fix it, should be one property.

Reply:
Done